### PR TITLE
#564 lg 03 07 interface characteristics

### PR DESCRIPTION
**Pull request by gernotstarke**
looking for feedback

### CODE REVIEW

File: documentation/decisions/008-specify-quality-requirements.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 008: Architects are able to specify quality requirements
+
+## Status: Accepted
+
+## Context 
+Within the FOUNDATION level working group we had long (aka _endless_) discussions about quality and quality requirements.
+Conclusion is:
+
+* Software architects shall be able to formulate precise quality requirements, i.e. as quality scenarios.
+* The shall not _invent_ these, but formulate in cooperation or collaboration with other stakeholders.
```
Comment (ulibecker) [inline documentation/decisions/008-specify-quality-requirements.md:10]: Typo: Should be "They shall"

File: documentation/decisions/008-specify-quality-requirements.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ 
+## Rationale:
+
+* The ability to formulate such requirements in precise and practically usable ways is part of many CPSA-Advanced level certifications, therefore it *must* be taught and practiced within foundation level trainings.
```
Comment (ulibecker) [inline documentation/decisions/008-specify-quality-requirements.md:14]: Would add:

* Often, there is a lack of good quality requirements in development projects. Due to the strong influence of quality requirements on the architecture, architects should be able to formulate them.

**Review by ulibecker — CHANGES_REQUESTED** [review]

File: documentation/decisions/008-specify-quality-requirements.md (hunk 2 above)
Comment (rhoadesre) [reply documentation/decisions/008-specify-quality-requirements.md:14]: @Ulibecker: This topic is different than the "LG 03-07 interface characteristics". 
Is it a change request for this issue or something else?

**Review by rhoadesre — CHANGES_REQUESTED** [review]
Please separate the two changes. 
I am fine with the resolution of the original issue (interface characteristics), but the documentation of quality requirements is a completely different issue. 
And I, like Daniel, definitely do not agree that a decision was made regarding this latter issue.

File: documentation/decisions/008-specify-quality-requirements.md (hunk 2 above)
Comment (rhoadesre) [reply documentation/decisions/008-specify-quality-requirements.md:14]: Refer to issues #15, #299 , #396 , #539 , and most recently #597.
There are several reasons (legal, etc.) why we shouldn't do this.

File: docs/03-design/LG-03-07.adoc [hunk 3]
```diff
@@ -4,20 +4,36 @@
 [[LG-03-07]]
 ==== LZ 03-07 [ehemaliges LZ 2-09]: Schnittstellen entwerfen und festlegen (R1-R3)
 
-Softwarearchitekt:innen kennen die hohe Bedeutung von Schnittstellen. Sie können Schnittstellen zwischen Architekturbausteinen sowie externe Schnittstellen zwischen dem System und Elementen außerhalb des Systems entwerfen bzw. festlegen.
+Softwarearchitekt:innen kennen die hohe Bedeutung von Schnittstellen.
```
Comment (alxlo) [inline docs/03-design/LG-03-07.adoc:7]:
```suggestion
Softwarearchitekt:innen erkennen und beachten die fundamentale Bedeutung von Schnittstellen. 
```
Die "hohe Bedeutung" klingtr für mich sehr gesteltzt und sperrig.

File: documentation/decisions/008-specify-quality-requirements.md (hunk 1 above)
Comment (gernotstarke) [reply documentation/decisions/008-specify-quality-requirements.md:10]: my fault, I accidently mixed two completely different topics. We need to take the ADR-008 out of THIS discussion and PR. Sorry, @rhoadesre, seems I got confused yesterday...

**Review by alxlo — APPROVED** [review]

**Review by rhoadesre — DISMISSED** [review]

**Comment by rhoadesre** [comment]
I'm fine with these changes, but the changes are blocked.
@Uli: is this because of your change request or mine?

**Review by sippsack — DISMISSED** [review]
Looks good. I like the ADR. 👍 (eat your own dog food)

**Comment by kraemerdr** [comment]
@sippsack The ADR has accidentally been introduced in **this** issue and belongs to #597 instead. ;-)
What exactly do you mean with 'eat your own dog food' in this case?

**Comment by gernotstarke** [comment]
It should better be called "drink your own champagne", a little less disgusting ...

I guess @sippsack referred to the ongoing recital of "use ADRs to document architectural decisions" - and we used an ADR ourselves to document OUR decision...

**Comment by gernotstarke** [comment]
shall I open a NEW PR with ONLY the interface-related change in it?
So it becomes better comparable to the existing version?

**Comment by sippsack** [comment]
@kraemerdr https://en.wikipedia.org/wiki/Eating_your_own_dog_food

I thought the ADR belongs to this PR. I didn't recognize that it belongs somewhere else.

**Comment by kraemerdr** [comment]
@sippsack I know the phrase, of course, but was wondering what the 'dog food' actually is here. If you meant ADRs in general, that makes perfect sense. ;-)

**Comment by gernotstarke** [comment]
I close this one, as the many changes are too difficult to track. I have opened a new PR #659 on a fresh branch

**Pull request closed without merging**